src/main/java/com/snowflake/kafka/connector/internal/streaming/schemaevolution/ColumnInfos.java

```diff
@@ -12,6 +12,11 @@ public ColumnInfos(String columnType, String comments) {
     this.comments = comments;
   }
 
+  public ColumnInfos(String columnType) {
```

Comment:
Why plural form here?

Reply:
you are the author of this class :D

Reply:
and this is constructor so it has to be class name

Reply:
so that's my mistake, sorry about that 😄